ubi7/Dockerfile.pgo-backrest.ubi7

```diff
@@ -2,8 +2,7 @@ ARG PREFIX
 ARG BASEVER
 FROM ${PREFIX}/pgo-base:ubi7-${BASEVER}
 
-LABEL name="crunchydata/pgo-backrest-" \
-    run='docker run -d -p 8080:80 --name=web-app web-app' \
+LABEL name="crunchydata/pgo-backrest" \
```

Comment:
Is there a reason this breaks from the naming pattern of the other files?

Reply:
Yes: I missed updating it.